cdap-ranger/README.rst

```diff
@@ -216,7 +215,158 @@ expect wildcard ``*`` to match 0 characters (absence of value) so a ``*`` should
 example to grant a user privilege on all ``programs`` the wildcard value should be as shown below.
 
 .. image:: _images/policy_management_wildcard.png
-:align: center
+  :align: center
+
+A Case Study
+============
```

Comment:
Maybe this ----------

Just like how it is for Conclusion section

Reply:
That is actually a top-level heading. Whereas Conclusion is a subsection of this.